src/mach/imports.rs

```diff
@@ -195,9 +216,9 @@ impl<'a> BindInterpreter<'a> {
 	            // throwBadBindingAddress(address, segmentEndAddress, segmentIndex, start, end, p);
 	            // (this->*handler)(context, address, type, symbolName, symboFlags, addend, libraryOrdinal, "", &last);
 	            // address += sizeof(intptr_t);
+                    imports.push(Import::new(&bind_info, libs, segments, start_of_sequence));
                     let seg_offset = bind_info.seg_offset.wrapping_add(ctx.size() as u64);
                     bind_info.seg_offset = seg_offset;
-                    imports.push(Import::new(&bind_info, libs, segments));
```

Comment:
Completing the example, `goblin` as of eab5060f39e88cc4e181afec3f71743c2fdb384f agrees with `dyldinfo -bind`, both showing:

```__DATA  __const          0x102EE7168    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
__DATA  __const          0x10304F8E8    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
__DATA  __const          0x10304F980    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
__DATA  __const          0x1030EA290    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
__DATA  __const          0x1030EA378    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
__DATA  __const          0x1030EA438    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
__DATA  __const          0x1030EA498    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
__DATA  __const          0x103113E88    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
__DATA  __data           0x10313B638    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
__DATA  __data           0x10313C200    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
__DATA  __data           0x10313D008    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
__DATA  __data           0x10313D040    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
__DATA  __data           0x10313D078    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
__DATA  __data           0x10313D0B0    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
```

`goblin` as of 150f92e590d697a0305fac3f39d2a2db79a00f56 shows:

```__DATA  __const          0x10304F8E8    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
__DATA  __const          0x10304F980    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
__DATA  __const          0x1030EA290    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
__DATA  __const          0x1030EA378    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
__DATA  __const          0x1030EA438    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
__DATA  __const          0x1030EA498    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
__DATA  __const          0x103113E88    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
__DATA  __data           0x10313B638    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
__DATA  __data           0x10313C200    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
__DATA  __data           0x10313D008    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
__DATA  __data           0x10313D0B0    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
__DATA  __data           0x10313D0B8    pointer      0 libc++           __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info
```

…which is a combination of the off-by-one I described and the missing repeated binds.